packages/back-office-subscribers/package.json

```diff
@@ -8,7 +8,8 @@
 		"test": "jest"
 	},
 	"dependencies": {
-		"axios": "^1.4.0"
+		"axios": "^1.4.0",
+		"lodash.pick": "^4.4.0"
```

Comment:
Not passionate about this but interesting if you want to avoid a dependency -

 https://youmightnotneed.com/lodash

Reply:
Thanks that's a good link for the bookmarks! The balance here is that I'm only depending on `lodash.pick` rather than importing the whole of `lodash`.

In this case, I think the plain JS implementation is pretty verbose so I do prefer the lodash way for this and have used it in a similar way in the api code

Reply:
Here's the comparison:

lodash:
```
const pick = require('lodash.pick');

module.exports = async (context, message) => {
	context.log(`invoking nsip-document function with message: ${JSON.stringify(message)}`);

	const document = pick(message, [
		'documentId',
		'caseId',
		'caseRef',
		'documentReference',
		'version',
		'examinationRefNo',
		'filename',
		'originalFilename',
		'size',
		'mime',
		'documentURI',
		'publishedDocumentURI',
		'virusCheckStatus',
		'fileMD5',
		'dateCreated',
		'lastModified',
		'caseType',
		'documentStatus',
		'redactedStatus',
		'publishedStatus',
		'datePublished',
		'documentType',
		'securityClassification',
		'sourceSystem',
		'origin',
		'owner',
		'author',
		'representative',
		'description',
		'stage',
		'filter1',
		'filter2'
	]);

	context.bindings.document = {
		...document,
		modifiedAt: new Date()
	};
};
```

plain js
```
module.exports = async (context, message) => {
  context.log(`invoking nsip-document function with message: ${JSON.stringify(message)}`);

  const {
    documentId,
    caseId,
    caseRef,
    documentReference,
    version,
    examinationRefNo,
    filename,
    originalFilename,
    size,
    mime,
    documentURI,
    publishedDocumentURI,
    virusCheckStatus,
    fileMD5,
    dateCreated,
    lastModified,
    caseType,
    documentStatus,
    redactedStatus,
    publishedStatus,
    datePublished,
    documentType,
    securityClassification,
    sourceSystem,
    origin,
    owner,
    author,
    representative,
    description,
    stage,
    filter1,
    filter2
  } = message;

  const document = {
    documentId,
    caseId,
    caseRef,
    documentReference,
    version,
    examinationRefNo,
    filename,
    originalFilename,
    size,
    mime,
    documentURI,
    publishedDocumentURI,
    virusCheckStatus,
    fileMD5,
    dateCreated,
    lastModified,
    caseType,
    documentStatus,
    redactedStatus,
    publishedStatus,
    datePublished,
    documentType,
    securityClassification,
    sourceSystem,
    origin,
    owner,
    author,
    representative,
    description,
    stage,
    filter1,
    filter2,
    modifiedAt: new Date()
  };

  context.bindings.document = document;
};
```

I think it's OK for a few fields but as it's a pretty massive list it's a bit verbose